scripts/travis_yml_test.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test to ensure travis.yml file and protractor.conf.js have the same test suites"""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import re
+
+from core.tests import test_utils
+import python_utils
+
+
+class TravisCIFileTests(test_utils.GenericTestBase):
```

Comment:
How is this test getting triggered? I think as its name *_test.py it must be running as a backend test, but it should be a part of the lint test, right?

Reply:
Sean suggested having a backend test [here](https://github.com/oppia/oppia/issues/8269#issuecomment-568591932).
So I change this to a lint test?

Reply:
@seanlip, Do you want this to be a unit test?  I think by `backend test` you mean the test should be written in python, right?

Reply:
It's a bit of a grey area, but I think a backend unit test is probably fine. The linting process is already a bit messy and I don't want to make it messier.